lib/players/computer.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../player.rb'
```

Comment:
Classes should not be requiring files, that should be handled by the environment.